cmd/opentelemetry/app/defaultconfig/default_config_test.go

```diff
@@ -200,7 +200,10 @@ func TestService(t *testing.T) {
 			test.cfg.Factories = factories
 			createDefaultConfig := test.cfg.DefaultConfigFactory(v)
 
-			cfg, err := createDefaultConfig(viper.New(), factories)
+			// command set flag is parsed into otelViper
+			otelViper := viper.New()
+			command.Flags().StringArray("set", []string{}, "set overrides settings in otelcol")
```

Comment:
Do we use `otelcol` in the other options? I'd prefer to use the full name here ("OpenTelemetry Collector") if we don't use "otelcol" yet in the user interface.

Reply:
I'll fix that